db/migrate/20201107233243_implement_database.rb

```diff
@@ -0,0 +1,43 @@
+class ImplementDatabase < ActiveRecord::Migration[6.0]
+  def change
+    create_table :bridges do |t|
+      t.string :name, null: false
+      t.string :inboundURL, null: false
+      t.string :outboundURL, null: false
```

Comment:
Database columns are typically snake case. These should be `inbound_url` & `outbound_url`